projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -225,7 +201,20 @@ function ( $event ) use ( $updated_at ) {
 	 * Clear the cron cache.
 	 */
 	public static function clear_cron_cache() {
-		wp_cache_delete( 'alloptions', 'options' );
+		wp_cache_delete( 'cron', 'options' );
+		wp_load_alloptions( true );
+	}
+
+	/**
+	 * Reload the cron cache in pre_schedule_event hook.
+	 *
+	 * @param string $event The event hook name.
+	 */
+	public static function clear_cron_cache_pre( $event ) {
+		// If the transient is set and an external event is about to run, it means that the cron cache must be refreshed.
+		if ( self::PLUGIN_CRON_HOOK !== $event && get_transient( 'pre_schedule_event_clear_cron_cache' ) ) {
```

Comment:
According to [the doc](https://developer.wordpress.org/reference/functions/wp_schedule_event/) It seems the first param is result and the second one is $event object.

Reply:
Yes thanks, changed.